### DISCUSSION TITLE
How to run a workflow when the PR title is changed?

### DISCUSSION THREAD

**Discussion by vadi2** [community/community#48695] [Actions] [answered]
### Select Topic Area

Product Feedback

### Body

We enforce a naming style for our PRs and would like to re-run an action when the PR title has been updated. I checked the docs, but didn't find an explicit event for this.

What's the best way of going about here?

**Comment by jsoref** [comment, marked as answer]
I believe this is what you want:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
```yml
on:
  pull_request:
    types: [edited]
```

There are a number of workflows that do what you're looking for, you should be able to find one and borrow the code.

**Reply by vadi2** [reply]
Thanks, that did it.

**Comment by yeikel** [comment]
Follow up for question about this : https://github.com/orgs/community/discussions/101695
